source/main.cc

```diff
@@ -450,6 +450,7 @@ int main (int argc, char *argv[])
 #ifdef ASPECT_USE_FP_EXCEPTIONS
   // enable floating point exceptions
   feenableexcept(FE_DIVBYZERO|FE_INVALID);
+  feclearexcept(FE_DIVBYZERO|FE_INVALID);
```

Comment:
Wait, aren't you first setting bits and then clearing them again? I don't think I quite understand how this is supposed to work. Same for the code in the cmake check...